src/patternfly/components/Switch/switch.scss

```diff
@@ -1,12 +1,12 @@
 .pf-c-switch {
   // Switch
-  --pf-c-switch--Height: calc(#{var(--pf-global--FontSize--md)} * #{var(--pf-global--LineHeight--md)});
-  --pf-c-switch--Width: calc(#{var(--pf-c-switch--Height)} * #{var(--pf-global--golden-ratio)});
+  --pf-c-switch--Height: calc(var(--pf-c-switch__label--FontSize) * var(--pf-c-switch--LineHeight));
+  --pf-c-switch--Width: calc(var(--pf-c-switch--Height) + #{pf-size-prem(2px)} + var(--pf-c-switch__toggle--before--Width));
   --pf-c-switch--LineHeight: var(--pf-global--LineHeight--md);
```

Comment:
Curious why we have `line-height` on `.pf-c-switch` and `font-size` on `.pf-c-switch__label`. Can we move them both to the same element, or is there a reason to keep them separate?

Reply:
Great point, moved the font-size from the label to the switch

Reply:
I did have to remove the font-size variable from the label, because now if the font-size changes the say 24px, the var set on the label overrides it to still be 16px. Is this a breaking change?

Reply:
I think it's fine, but could you add the var and declaration back and just make the value `--pf-c-switch__label--FontSize: var(--pf-c-switch--FontSize)`?